src/test/scala/mesosphere/marathon/MarathonTestHelper.scala

```diff
@@ -110,7 +110,7 @@ object MarathonTestHelper {
       .addResources(memResource)
       .addResources(diskResource)
 
-    portsResource.foreach(offerBuilder.addResources(_))
+    portsResource.foreach(offerBuilder.addResources)
```

Comment:
why? It's not related to this patch and I don't think that we have a team agreement what we like better.